torch/utils/bottleneck/__main__.py

```diff
@@ -183,9 +184,9 @@ def main():
     scriptfile = args.scriptfile
     scriptargs = [] if args.args is None else args.args
     scriptargs.insert(0, scriptfile)
-    cprofile_sortby = 'tottime'
+    cprofile_sortby = os.environ.get("CPROFILE_SORTBY", 'tottime')
```

Comment:
what is the rationale for making these two configs configurable via env var versus CLI args? If the requirement is that this is env var configurable, consider adding this as argument as well like so:

```
parser.add_argument("--cprofile_sortby", default=os.environ.get("CPROFILE_SORTBY", "tottime"))
```